VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
Yea, I think you're right that the documentation and api version might want to live in a separate repository.